static/js/components/SubscriptionsList.js

```diff
@@ -14,19 +16,54 @@ const channelClassName = (channelName, currentChannel) =>
   currentChannel === channelName ? "location current-location" : "location"
 
 export default class SubscriptionsList extends React.Component<Props> {
+  makeChannelLink = R.curry((channel: Channel) => {
```

Comment:
does this need to be a curried function? it's only got one argument

Reply:
No, at some point I wanted to pass in an extra parameter for whether the channel is moderated by the user or not, but then I remembered I could use `channel.user_is_moderator` so it was unnecessary. I'll remove it